pkgs/development/python-modules/sip/4.x.nix

```diff
@@ -39,11 +45,6 @@ buildPythonPackage rec {
 
   enableParallelBuilding = true;
 
-  pythonImportsCheck = [
-    sip-module
-    "sipconfig"
-  ];
-
   doCheck = true;
```

Comment:
shouldn't we add `unittestCheckHook` to `nativeCheckInputs`? (See
https://github.com/Python-SIP/sip/blob/main/test/utils/sip_test_case.py )

Reply:
That repo is for the newer sip6 only. Adding this to sip4 results in no tests being run.

If you download the src tar file directly for this sip4 package, there is no test directory at all.